File: csharp/Auction-TestDesign-Kata/Test-Auction/AuctionMessageTranslatorTest.cs
using Auction;

namespace Test_Auction_TestDesign_Kata;

public class AuctionMessageTranslatorTest
{
    [Fact]
    public void NotifiesAuctionClosedWhenCloseMessageReceived()
    {
        var message = "SOLVersion: 1.1; Event: CLOSE;";
        // TODO: write a test for this message translation
    }

    [Fact]
    public void NotifiesBidDetailsWhenPriceMessageReceived()
    {
        var message = "SOLVersion: 1.1; Event: PRICE; CurrentPrice: 192; Increment: 7; Bidder: Someone else;";
        // TODO: write a test for this message translation
    }
}

File: kotlin/src/test/kotlin/org/sammancoaching/AuctionMessageTranslatorTest.kt
package org.sammancoaching

import kotlin.test.Test

class AuctionMessageTranslatorTest {
    @Test
    fun notifiesAuctionClosedWhenCloseMessageReceived() {
        val message = "SOLVersion: 1.1; Event: CLOSE;"
        // TODO: write a test for this message translation
    }

    @Test
    fun notifiesBidDetailsWhenPriceMessageReceived() {
        val message = "SOLVersion: 1.1; Event: PRICE; CurrentPrice: 192; Increment: 7; Bidder: Someone else;"
        // TODO: write a test for this message translation
    }
}